openlibrary/macros/LocateButton.html

```diff
@@ -0,0 +1,9 @@
+$def with(edition_key=None)
+
+$ locateUrl = "/books/XXX/-/borrow?action=locate".replace('XXX', edition_key or '')
+
+<div class="cta-button-group">
+  <a class="cta-btn cta-btn--available" href="$locateUrl">
```

Comment:
Needs analytics clicktracking

```suggestion
  <a class="cta-btn cta-btn--available" href="$locateUrl" data-ol-link-track="CTAClick|Locate">
```